core/src/main/java/org/infinispan/util/concurrent/locks/impl/DefaultLockManager.java

```diff
@@ -56,6 +62,8 @@
    @Inject private Configuration configuration;
    @Inject @ComponentName(KnownComponentNames.TIMEOUT_SCHEDULE_EXECUTOR)
    private ScheduledExecutorService scheduler;
+   @Inject @ComponentName(KnownComponentNames.REMOTE_COMMAND_EXECUTOR)
```

Comment:
:-1: 
use the `ASYNC_OPERATIONS_EXECUTOR`. 
The `REMOTE_COMMAND_EXECUTOR` will probably be removed in the future.